test-global-distribution.html

```diff
@@ -1,45 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
-    <title>Linking Required Peer JavaScript Dependencies</title>
-</head>
-
-<body>
-    <!--- Must reference peer dependencies here first since these are not included in wrapped global package -->
-    <script src="https://unpkg.com/lodash@4.9.0/lodash.js"></script>
-    <script src="https://unpkg.com/immutable@3.8.2/dist/immutable.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/i18next/19.6.3/i18next.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/axios/0.20.0/axios.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/humanize-plus/1.8.2/humanize.min.js"></script>
-
-    <!-- Reference the most recently built global distribution package -->
-    <script src="./dist-global/index.js"></script>
-
-    <script>
-        // Testing 'immutable' peer dependency usage
-        AndcultureCode.CollectionUtils.length([1, 2]);
-
-        // Testing 'Lodash' peer dependency usage
-        AndcultureCode.CollectionUtils.difference([1, 2], [1]);
-
-        // Testing 'i18next' peer dependency usage
-        AndcultureCode.LocalizationUtils.initialize({ type: "backend", init() { }, }, [{
-            code: "en_US",
-            resources: null,
-        }]);
-        AndcultureCode.LocalizationUtils.changeCultureCode('en-UK');
-
-        // Testing 'query-string' peer dependency usage
-        AndcultureCode.RouteUtils.queryStringToObject('#token=bada55cafe');
-
-        // Testing 'axios' peer dependency usage
-        AndcultureCode.ServiceUtils.configure();
-
-        // Testing 'humanize-plus' peer dependency usage
-        AndcultureCode.StringUtils.pluralize(3, 'duck');
-    </script>
-</body>
-
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="utf-8" />
+        <title>Linking Required Peer JavaScript Dependencies</title>
+    </head>
+
+    <body>
+        <!--- Must reference peer dependencies here first since these are not included in wrapped global package -->
+        <script src="https://unpkg.com/lodash@4.9.0/lodash.js" />
+        <script src="https://unpkg.com/immutable@3.8.2/dist/immutable.js" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/i18next/19.6.3/i18next.min.js" />
+        <script src="https://unpkg.com/browse/i18next-browser-languagedetector@6.0.1/dist/umd/i18nextBrowserLanguageDetector.min.js" />
```

Comment:
Prettier auto-formatted the file, but this is the only new line. This package is now required for LocalizationUtils to properly load up.